Disaster Recovery/SafeHaven 5 CPCVM to CLC/create-linux-dccf-clc.md

```diff
@@ -10,7 +10,7 @@
 This article explains how to create a Linux Protection Group.
 
 ### Requirements
-Access to the Production SRN in CenturyLink Cloud and permissions to add raw disks to it.
+Access to the Production SRN in CPCVM (DCCF) and permissions to add raw disks to it.
```

Comment:
I think this part of the KB refers to CLC (CenturyLink Cloud) and not the former DCCF product. I would change this to CLC.